dm-manager/pkg/device/device.go

```diff
@@ -245,7 +243,8 @@ func (dc *Controller) shouldSyncPowerStatus(invHost *computev1.HostResource) boo
 
 func (dc *Controller) shouldHandlePowerChange(invHost *computev1.HostResource) bool {
 	return invHost.GetCurrentAmtState() == computev1.AmtState_AMT_STATE_PROVISIONED &&
-		invHost.GetDesiredPowerState() != invHost.GetCurrentPowerState()
+		(invHost.GetDesiredPowerState() != invHost.GetCurrentPowerState() ||
+			invHost.GetDesiredPowerState() == computev1.PowerState_POWER_STATE_RESET)
```

Comment:
Is there a PR in API as well ? Else this condition would always trigger after a power_state_reset is set via user action. The manager cannot update the desired state.

Reply:
API changes is not required in a separate PR to handle the double reset operation, but this condition has no effect. I will remove it.